### PR TITLE
Simplify stub code.

### LINKED ISSUE #91: Should we simplify stub code?
Right now they look like:

```
    def add(self, other, axis='columns', level=None, fill_value=None):
        """A stub for the equivalent method to `pd.DataFrame.add()`.
        The method `pd.DataFrame.add()` is not implemented yet.
        """
        raise PandasNotImplementedError(class_name='pd.DataFrame', method_name='add')
```

Should we simplify it to something like

```
def add(**kwargs):
  unsupported_function()
```

or

```
add = unsupported_function()
```

or even

```
unsupported_function('add')
```

Reason I think we should simplify:

1. I don't see a strong need to show all the arguments, in addition to the function name. (Correct me if I'm wrong).
2. Perception wise, it'd be better to have a shorter missing file than a very long file.

### PR DESCRIPTION
**Pull request by ueshin**
Closes #91.

### CODE REVIEW

File: dev/_make_missing_functions.py [hunk 1]
```diff
@@ -217,6 +217,7 @@ def _main():
         print('MISSING functions for {}'.format(original_type.__name__))
         for name, signature in missing:
             print(make_misssing_function(original_type, name, signature))
+            # print("""    {0} = unsupported_function('{0}')""".format(name))
```
Comment (rxin) [inline dev/_make_missing_functions.py]: why is this commented out?

**Comment by rxin** [comment]
LGTM other than that one comment. Please merge if tests pass.

File: dev/_make_missing_functions.py (hunk 1 above)
Comment (ueshin) [reply dev/_make_missing_functions.py]: ah, not a special reason but only one of the two will be needed.

**Comment by ueshin** [comment]
I'd merge this now. Thanks!

**Pull request merged**